trl/trainer/ppo_trainer.py

```diff
@@ -142,10 +142,10 @@ class PPOTrainer(BaseTrainer):
 
     def __init__(
         self,
-        config: Optional[PPOConfig] = None,
-        model: Optional[PreTrainedModelWrapper] = None,
+        config: PPOConfig = None,
+        model: PreTrainedModelWrapper = None,
```

Comment:
If the default value is `None`, the type must be `Optional[]`; please see #1297 and https://github.com/hauntsaninja/no_implicit_optional.

(Repeated for all instances in this PR.)

Reply:
Fixed